polkadot/xcm/xcm-simulator/fuzzer/src/fuzz.rs

```diff
@@ -155,6 +158,34 @@ fn run_input(xcm_messages: [XcmMessage; 5]) {
 	println!();
 
 	for xcm_message in xcm_messages {
+		fn matches_blocklisted_messages(message: Instruction<()>) -> bool {
+			matches!(message, Transact { .. })
+		}
+		// We check XCM messages recursively for blocklisted messages
+		fn matches_recursive(message: &mut Instruction<()>) -> Vec<Instruction<()>> {
```

Comment:
Can you pull these two functions out maybe?

Reply:
Should be addressed